extensions/Cake.LittleHelpers.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.LittleHelpers
+NuGet: Cake.LittleHelpers
```

Comment:
Repo doesn't seem to be anywhere on GitHub... Blacklist

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194